test/e2e/wdio.local.conf.js

```diff
@@ -1,7 +1,7 @@
 const baseConf = require('./wdio.base.conf')
 
 // https://sites.google.com/a/chromium.org/chromedriver/downloads
-const CHROME_DRIVER_VERSION = '79.0.3945.36'
+const CHROME_DRIVER_VERSION = '81.0.4044.69'
```

Comment:
It seems that version is also defined on the Dockerfile. 
Could we define it once and then replace it in all the files?

Reply:
Driver and chrome version are not the same.
Not sure how we could mutualize that if it was.

Reply:
OK then 😄